tests/safeds/data/tabular/containers/_table/test_replace_column.py

```diff
@@ -47,26 +48,35 @@
             ),
         ),
     ],
+    ids=["multiple Columns", "one Column"],
 )
-def test_should_replace_column(table: Table, column_name: str, column: Column, expected: Table) -> None:
-    result = table.replace_column(column_name, column)
-    assert result.schema == expected.schema
```

Comment:
Generally, we decided that we want these schema checks for better error messages.